src/docs/Updates/Roadmap/index.mdx

```diff
@@ -39,12 +37,20 @@ description: 'Discover what we are working on right now'
 
 **Add new pattern:**
 
+- Breadcrumb
 - Price tag
```

Comment:
I think this is way too ambitious, we're already late on some patterns from sprint 3

Reply:
Yes, I postpone the breadcrumb to sprint 5